kentik_api_library/kentik_api/requests_payload/custom_applications_payload.py

```diff
@@ -51,9 +51,10 @@ class GetAllResponse(List[GetResponse]):
     @classmethod
     def from_json(cls, json_string: str):
         dic = json.loads(json_string)
+        #        dic = from_json(cls.__name__, json_string)  # using this line causes mypy error
```

Comment:
What error does mypy return? Are we planning to do something with it?

Reply:
It raises error: Argument 2 to "from_dict" has incompatible type "str"; expected "Dict[str, Any]" at 57 line.

Reply:
I guess we need some kind of type assertion here to satisfy the  Mypy. @mateuszmidor Any ideas for that?

@lukaa12 Please remove commented out code, resolve the issue or try to suppress the linter and leave a TODO.

Reply:
I checked it and the dic is actually a list, but mypy thinks it is a dictionary and that is why it raises error here.

Reply:
If `dic` is a list, then please name it as e.g. `list` or `data` instead of adding a comment, that it is a list.
"Argument 2 to "from_dict" has incompatible type "str";" means that `item` has type `str` but with proper payload it should be `Dict[str, Any]`.

Reply:
The code is fine now. Resolved